examples/web-wagmi/src/App.tsx

```diff
@@ -72,7 +72,6 @@ const client = createClient({
 const lensConfig: LensConfig = {
   bindings: wagmiBindings(),
   environment: staging,
-  sources: [sources.lenster, sources.orb, 'any-other-app-id'],
```

Comment:
Removed it for now that [T-4762](https://avara.height.app/T-4762) is not implemented yet as it won't show publications created from within the example app.